YunoHost/yunohost
# Get rid of etckeeper

## The problem

etckeeper has been added with no actual motivation and we still don't really use it.

But it does create several issues with user encountering cases where etckeeper complains because no identity is defined, or it display huge boring log about files being versionned when you run apt, or the git index ends up being broken for weird reason which leads to other issues, or somehow the git history ends up being several Go's big.

## Solution

This was a big YAGNI. Get rid of it.

## PR Status

Yolo commited

## How to test

Test a new install with this, and check if stuff goes okay on previous instances

## Validation
